gsa/src/web/pages/tickets/dashboard/statusdisplay.js

```diff
@@ -0,0 +1,181 @@
+/* Copyright (C) 2019 Greenbone Networks GmbH
+ *
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+import React from 'react';
+
+import {scaleOrdinal} from 'd3-scale';
+
+import {_, _l} from 'gmp/locale/lang';
+
+import Filter, {TICKETS_FILTER_FILTER} from 'gmp/models/filter';
+import FilterTerm from 'gmp/models/filter/filterterm';
+import {TICKET_STATUS} from 'gmp/models/ticket';
+
+import {isDefined} from 'gmp/utils/identity';
+
+import DonutChart from 'web/components/chart/donut';
+
+import {registerDisplay} from 'web/components/dashboard/registry';
+
+import DataDisplay from 'web/components/dashboard/display/datadisplay';
+import {
+  renderDonutChartIcons,
+} from 'web/components/dashboard/display/datadisplayicons';
+import {
+  percent,
+} from 'web/components/dashboard/display/utils';
+import withFilterSelection from 'web/components/dashboard/display/withFilterSelection';  // eslint-disable-line max-len
+
+import PropTypes from 'web/utils/proptypes';
+import Theme from 'web/utils/theme';
+
+import {TicketsListLoader} from './loaders';
+
+const taskStatusColorScale = scaleOrdinal()
```

Comment:
copy/paste error here, I guess ;) It should be ```ticketStatusColorScale```

Reply:
Fixed